examples/invoke-simple/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.7-buster
```

Comment:
Can the image be 3.7-slim-buster?

https://hub.docker.com/layers/python/library/python/3.7-slim-buster/images/sha256-38f7320f8b7fd0e635f3bb298a43bb47c07de5f0e2df059a2a55b7ff197dedf0?context=explore

and does pip work with a slim buster image?

Reply:
it works.